server/conf/messages

```diff
@@ -49,7 +55,7 @@ validation.errorAnnouncementSr=There are errors in the form. Please fix before c
 content.requiredFieldsAnnotation=Note: Fields marked with a * are required.
 
 # Message displayed below "Choose File" button to say that on the phone users can use their camera.
-content.mobileFileUploadHelp=On your phone? “Choose File” also allows you to use your phone camera to upload documents.
+content.mobileFileUploadHelp=On your phone? ???Choose File??? also allows you to use your phone camera to upload documents.
```

Comment:
Looks like the curly quotes didn't come through properly, but we should've been using regular quotes anyway. Can we change to regular `"` quotes?

Reply:
Instead of reverting the diff, could you change them to straight quotes, since you're in here anyway?

Reply:
Done.